pkg/api/azenvtypes.go

```diff
@@ -138,18 +138,18 @@ var (
 	// AKSWindowsServer2019OSImageConfig is the aks-engine image based on Windows Server 2019
 	AKSWindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-smalldisk-2109",
+		ImageSku:       "2019-datacenter-core-smalldisk-2112",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.2213.210927",
+		ImageVersion:   "17763.2300.211204",
```

Comment:
It's expected that we built this VHD a day after the new containerd VHD?

Reply:
Yes, the azure devops pipeline for the docker VHD build had a flake and I triggered it the next day :(
That timestamp is the day the pipelines to build the VHD were run...